helm/aws-ebs-volume-tagger/templates/rbac.yaml

```diff
@@ -0,0 +1,38 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+rules:
+  - apiGroups: 
+    - ""
+    resources: 
+    - "persistentvolumes"
+    verbs: 
+    - "get"
+    - "list"
+    - "watch"
+  - apiGroups: 
+    - ""
+    resources: 
+    - "configmaps"
+    verbs: 
```

Comment:
Apparently you don't need all these permissions. just `list` for persistentvolumes` and `get` for configmaps.
You could also restrict the configmap get to a specific name once you're settled with the `test` name change

Reply:
you're right, this is a good idea ❤️ .